_posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "This Week in RustFest 5: Announcing the venue, more awesome sponsors and first CfP speaker confirmed"
+socialImageSrc: '/assets/posts/twirf-6/new-sponsors.png'
+---
+
+Oh, what a great week for RustFest!
+
+## Venue, in the center of Berlin
+
+Earlier this week really great news arrived us: **our most anticipated venue confirmed they will host RustFest 2016!**
+
+At Unter den Linden 17, just steps from to the Brandenburg Gate, **Microsoft Berlin** entertains an amazing conference and event venue in an iconic corner building:
+
+![RustFest Venue 2016](/assets/posts/twirf-6/venue-house.png)
+
+
+We got the pleasure to visit the "Atrium", their newly equipped conference setting, a few weeks ago and directly knew: this is where we want to host RustFest. I mean, just look at it. And that picture doesn't even cover all the amazing space we're gonna have. Check out [their website to get some impressions](https://www.microsoft-berlin.de/de-de/berlin/).
+
+
+![RustFest Venue 2016](/assets/posts/twirf-6/venue-room.png)
+
+
+But wait, there's more. Microsoft Berlin was so impressed with RustFest and what we are up to, they decided to sponsor the event. WHAAA? Yes, Microsoft Berlin let's use the venue _for both days_ free of charge! Our dreams come true.
+
+You know what that means? Well, for once, **WE WILL HAVE WORKSHOPS**. With the venue being sponsored by awesome Microsoft, we have at least one awesome workshop room right on side. Further more we will convert the big room we have for talks on the first day to a big meet-n-greet-n-hack area during the second. So for everyone, who's just happy to finally meet other community members in person and just want to hang out and chat, we'll have space for that, too. Oh, my. This is shaping up to be quite awesome!
+
+
+## More sponsors
+
+Aside from [Microsoft Berlin](https://www.microsoft-berlin.de/de-de/berlin/), two other organisations decided to join the list of sponsors of RustFest this week. The always awesome [Mozillians](https://www.mozilla.org/en-US/) and [QiTASC](http://www.qitasc.at/), a Quality Assurance Provider for mobile from Vienna, decided to join the ranks. The website has been updated accordingly.
```

Comment:
Before seeing the link target I wondered who you refer to as "Mozillians". I think I'd rather go with "Mozilla" instead. The sentence would still work I think. :)